cli/script/command-executor.ts

```diff
@@ -785,7 +785,7 @@ function getPackageMetricsString(packageObject: PackageWithMetrics): string {
     var percentString: string = (activePercent === 100.0 ? "100" : activePercent.toPrecision(2)) + "%";
```

Comment:
Do you want to just make the exception for "0%" to display that instead of "0.0%" too?